pkgs/development/libraries/mesa/default.nix

```diff
@@ -137,6 +137,9 @@ self = stdenv.mkDerivation {
     "-Dgbm-backends-path=${libglvnd.driverLink}/lib/gbm:${placeholder "out"}/lib/gbm"
   ] ++ lib.optionals stdenv.isLinux [
     "-Dglvnd=true"
+
+    # Enable RT for Intel hardware
+    "-Dintel-clc=enabled"
```

Comment:
Should this be behind `enableOpenCL`?

Reply:
No, this is a dependency for RT, which uses compute shaders on Intel cards.